### PR TITLE
Simplify networking

### PR DESCRIPTION
**Pull request by applmak**

### CODE REVIEW

File: server/modules/module.js [hunk 1]
```diff
@@ -19,11 +19,11 @@ import * as wallGeometry from '../util/wall_geometry.js';
 import * as moduleTicker from './module_ticker.js';
 import assert from '../../lib/assert.js';
 import library from './module_library.js';
-import network from '../network/network.js';
+import * as network from '../network/network.js';
 import {StateManager} from '../state/state_manager.js';
 import {delay} from '../../lib/promise.js';
 import {getGeo} from '../util/wall_geometry.js';
-import {clients} from '../network/clients.js';
+import {clients} from '../network/network.js';
```
Comment (jmegq) [inline server/modules/module.js]: It this kind of double-import an idiom now, or should it be const clients = network.clients?

File: server/network/network.js [hunk 2]
```diff
@@ -55,67 +47,87 @@ class ClientInfo {
...
-network.close = function() {
-  io.close();
-  io = undefined;
-};
-
-// Sends a message to all clients.
-network.broadcast = function(msg, data) {
-  io.emit(msg, data);
-};
+  // Set up a timer to send the current time to clients every 10 seconds.
```
Comment (jmegq) [inline server/network/network.js]: I still don't get this... I mean, just sync the clocks with ntp, yo?
I can set that up.
Or is this for some other reason?

File: server/modules/module.js (hunk 1 above)
Comment (applmak) [reply server/modules/module.js]: "Destructuring" import is the right way to do it.

File: server/network/network.js (hunk 2 above)
Comment (applmak) [reply server/network/network.js]: I just want time to be monotonically increasing, and only performance.now / process.hrtime have such guarantees. The problem is that they don't share a zero. I *could* implement this as a single bit of data in the handshake, I suppose.

**Pull request merged**